src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+
+set BridgeHost=
+cd %2
+REM ensure bridge is stopped
+call stopbridge.cmd
+REM remove bin directory to ensure a rebuild
+del /s /q bin
+del *.rsp
```

Comment:
Why not 'git clean -xdf' instead of deleting bin and *.rsp?  If this script does a git pull, there is a good chance it may find newer package dependencies.  So I think it is cleaner to delete all the temporary build artifacts and ensure a clean state of the enlistment.